src/show/pages/public/Project/index.js

```diff
@@ -80,17 +80,14 @@ const Project = () => {
               {project?.amount_invested}$ / {project?.goal}$
             </div>
             <div className='px-5'>
-              <ProgressBar
-                percentage={
-                  (`${project?.amount_invested}` / `${project?.goal}`) * `100`
-                }
-              />
+              <ProgressBar percentage={project.achieved_goal_percentage} />
             </div>
             <div className='flex items-center justify-center mt-5'>
               <Button
                 className='flex w-1/6 justify-center'
-                variant='invert-outline'
+                variant='invertoutline'
```

Comment:
Do `inverted-outline` as your button variant, not `invertedoutline`.

Reply:
nu functioneste with -